testfiles/output/Book_pdfx1a.pdf_XmlUnitExpectedOutput.xml

```diff
@@ -1,44 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.6.0-SNAPSHOT" timestamp="1/11/23, 10:59 AM">
+<fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.6.0-SNAPSHOT" timestamp="7/14/22, 4:31 PM">
   <identification>
     <identity format="PDF/X" mimetype="application/pdf" toolname="FITS" toolversion="1.6.0-SNAPSHOT">
       <tool toolname="Droid" toolversion="6.5.2" />
-      <tool toolname="Jhove" toolversion="1.26.1" />
-      <tool toolname="Exiftool" toolversion="12.50" />
-      <tool toolname="Tika" toolversion="2.6.0" />
-      <version toolname="Jhove" toolversion="1.26.1">1a:2003</version>
+      <tool toolname="Jhove" toolversion="1.26.0" />
+      <tool toolname="Exiftool" toolversion="12.40" />
+      <tool toolname="Tika" toolversion="2.3.0" />
+      <version toolname="Jhove" toolversion="1.26.0">1a:2003</version>
       <externalIdentifier toolname="Droid" toolversion="6.5.2" type="puid">fmt/157</externalIdentifier>
     </identity>
   </identification>
   <fileinfo>
-    <size toolname="Jhove" toolversion="1.26.1">962199</size>
-    <creatingApplicationName toolname="Jhove" toolversion="1.26.1">Adobe PDF Library 10.0.1/Adobe InDesign CS6 (Windows)</creatingApplicationName>
-    <lastmodified toolname="Exiftool" toolversion="12.50">2015-08-20T22:29:04Z</lastmodified>
-    <created toolname="Exiftool" toolversion="12.50">2015-08-20T22:29:00Z</created>
-    <filepath toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">/fits/testfiles/input/Book_pdfx1a.pdf</filepath>
+    <size toolname="Jhove" toolversion="1.26.0">962199</size>

```

Comment:
This is looking much better! However, I'm surprised to see differences like these crop up. Did you run the tests with the overwrite flag?

Reply:
No, I did not run with that flag because I didn't want to update the expected output after resolving merge conflicts.
Time to finally accept this PR?

Reply:
Running the overwrite should be fine and should reduce the number of spurious differences in this PR. That was the point of creating the baseline.